src/audio/module_adapter/module/volume/volume_generic.c

```diff
@@ -26,7 +22,11 @@
 #include <stddef.h>
 #include <stdint.h>
 
-LOG_MODULE_DECLARE(volume, CONFIG_SOF_LOG_LEVEL);
+LOG_MODULE_DECLARE(volume_generic, CONFIG_SOF_LOG_LEVEL);
+
+#include <sof/audio/volume.h>
+
+#ifdef CONFIG_GENERIC
```

Comment:
doesn't need #endif?

Reply:
Its not a new block, #ifdef just shifted a few lines, #endif untouched